### PR TITLE
Update lessons for OpenRefine version 3.4

### PR DESCRIPTION
**Pull request by mariadelmarq**
Change to 01-introduction addresses issue #63 .

Changes to 02-working-with-openrefine address issues #49 , #52 , and #73 ; in addition to changes in trimming leading and trailing whitespaces upon data import.

The rest address updates to OpenRefine present in version 3.4.

### CODE REVIEW

**Comment by mariadelmarq** [comment]
@brownsarahm: so sorry to bug you, but I'll be teaching this lesson in a week. Is it possible for the PR to be merged before then, or should I plan for the materials to be different from what learners will be seeing?

Thanks so much for your work on this!

**Comment by brownsarahm** [comment]
Hi @mariadelmarq I've requested the other maintainers to review this.  I'm swamped right now.  However, you don't need to teach from the official version, you can teach from your own updated notes, since the learners generally follow you, not the materials.

**Comment by mariadelmarq** [comment]
@brownsarahm No worries, I completely understand! Ahhhh, do you mean build a local site with `make serve` and share my screen with learners? (Sorry for the noobie question, this is my first time teaching a carpentries lesson 🙈). I'm having issues with nokogiri, the install succeeds, but then `make serve` complains it can't install it. Any tips?

@gtlaflair and @isawil : please let me know if there is anything I can do to expedite this/help!

And thanks heaps to all of you for your work 😊

**Comment by brownsarahm** [comment]
*::puts instructor trainer hat on instead of maintainer hat::* You should be **demonstrating** openrefine while you're teaching. Typically instructors do not display the lesson material during the workshop to simplify the screen space.  For example, I don't give the link to the lesson materials until the end of the workshop, as a "follow up" resource rather than a teaching aid.

File: _episodes/02-working-with-openrefine.md [hunk 1]
```diff
@@ -223,13 +221,13 @@ It's common while exploring and cleaning a dataset to discover after you've made
 
 ## Trim Leading and Trailing Whitespace
 
-Words with spaces at the beginning or end are particularly hard for we humans to tell from strings without, but the blank characters will make a difference to the computer. We usually want to remove these. OpenRefine provides a tool to remove blank characters from the beginning and end of any entries that have them.
+Words with spaces at the beginning or end are particularly hard for we humans to tell from strings without, but the blank characters will make a difference to the computer. We usually want to remove these. As of version 3.4 of OpenRefine, the option to trim leading and trailing whitespaces is present at the moment of importing the data (see image at the top of this page). 
+
```
Comment (gtlaflair) [inline _episodes/02-working-with-openrefine.md:225]: Is this `+` supposed to be here? It looks like it might be a bullet point without any text after it?

**Review by gtlaflair — APPROVED** [review]
Looks good!

**Pull request merged**